ouroboros-consensus/src/Ouroboros/Consensus/Node/ExitPolicy.hs

```diff
@@ -0,0 +1,31 @@
+module Ouroboros.Consensus.Node.ExitPolicy
+  ( NodeToNodeClientResult (..)
+  , returnPolicy
+  -- Re-exports
+  , ReturnPolicy
+  ) where
+
+import           Ouroboros.Network.ExitPolicy
+import           Ouroboros.Consensus.MiniProtocol.ChainSync.Client
+
+
+-- | Result of any of the `node-to-node` mini-protocols.  We ignore all but
+-- `chain-sync` results.
+--
+data NodeToNodeClientResult =
```

Comment:
`NodeToNodeClientResult` is a little worrying, since we have NTN and NTC as a dichotomy. I think this name will be easy to misread/misunderstand.

Is there an alternative we could use for "Client". Does `NodeToNodeInitiatorResult` make any sense, for example?

Reply:
I agree `NodeToNodeInitiatorResult` sounds good to me.